src/ameba/runner.cr

```diff
@@ -160,9 +160,9 @@ module Ameba
 
       File.write(source.path, source.code) unless corrected_issues.empty?
     ensure
-      missing_location = Crystal::Location.new(nil, 0, 0)
+      missing_location = Crystal::Location.new(nil, 0, 0).to_s
       source.issues.sort_by! do |issue|
-        issue.location || missing_location
+        issue.location.to_s || missing_location
```

Comment:
if `issue.location` is `nil`, won't `.to_s` convert it into an empty string and `missing_location` is never used?

Reply:
yup, did fix it.